### ISSUE TITLE
Geeklog Package - Removed Files List is incorrect

### ISSUE THREAD

**Issue by eSilverStrike** [Geeklog-Core/geeklog#1126]
Ran the install package using the latest code in the repository and I notice that the **removed-files** list seems to be off. It lists 31 items including:

cron-emailgeeklogdailydigest
plugins/links/language/persian_utf-8.php
plugins/staticpages/language/persian_utf-8.php

The Persian language files are actually new files.
Also the emailgeeklogstores file was removed and replaced with cron-emailgeeklogdailydigest so it is using the wrong file name here

Also we dropped a lot of language files in Geeklog this version and none of them seem to be listed.

The **changed-files** file looks to be correct but I haven't really double checked much in it.

**Comment by eSilverStrike** [comment]
Here is the full removed file list that was generated for commit a78999566

cron-emailgeeklogdailydigest
plugins/links/language/persian_utf-8.php
plugins/staticpages/language/persian_utf-8.php
public_html/editors/ckeditor/samples/toolbarconfigurator/lib/codemirror/LICENSE
public_html/layout/denim/article/archivearticlebodytext.thtml
public_html/layout/denim/article/archivearticletext.thtml
public_html/layout/denim/article/articlebodytext.thtml
public_html/layout/denim/article/articletext.thtml
public_html/layout/denim/article/featuredarticlebodytext.thtml
public_html/layout/denim/article/featuredarticletext.thtml
public_html/layout/denim_three/article/archivearticlebodytext.thtml
public_html/layout/denim_three/article/archivearticletext.thtml
public_html/layout/denim_three/article/articlebodytext.thtml
public_html/layout/denim_three/article/articletext.thtml
public_html/layout/denim_three/article/featuredarticlebodytext.thtml
public_html/layout/denim_three/article/featuredarticletext.thtml
public_html/layout/modern_curve/article/archivearticlebodytext.thtml
public_html/layout/modern_curve/article/archivearticletext.thtml
public_html/layout/modern_curve/article/articlebodytext.thtml
public_html/layout/modern_curve/article/articletext.thtml
public_html/layout/modern_curve/article/featuredarticlebodytext.thtml
public_html/layout/modern_curve/article/featuredarticletext.thtml
public_html/vendor/jquery-ui/jquery-ui-dialogoptions.js
public_html/vendor/jquery-ui/jquery-ui-slideraccess.min.js
public_html/vendor/jquery-ui/jquery-ui-timepicker-addon-i18n.min.js
public_html/vendor/jquery-ui/jquery-ui-timepicker-addon.min.js
system/classes/Database/DbMysqli.php
system/classes/Database/DbPgsql.php
system/vendor/google/recaptcha/src/ReCaptcha/RequestMethod/Socket.php
system/vendor/mobiledetect/mobiledetectlib/LICENSE
system/vendor/pear/net_dns2/tests/Tests_Net_DNS2_ParserTest.php

**Comment by mystralkk** [comment]
Fixed with change set ac6d364

**Comment by eSilverStrike** [comment]
Just reran the install build process. The removed files list looks like this:

```
cron-emailgeeklogdailydigest
plugins/links/language/persian_utf-8.php
plugins/staticpages/language/persian_utf-8.php
public_html/editors/ckeditor/samples/toolbarconfigurator/lib/codemirror/LICENSE
public_html/layout/denim/article/archivearticlebodytext.thtml
public_html/layout/denim/article/archivearticletext.thtml
public_html/layout/denim/article/articlebodytext.thtml
public_html/layout/denim/article/articletext.thtml
public_html/layout/denim/article/featuredarticlebodytext.thtml
public_html/layout/denim/article/featuredarticletext.thtml
public_html/layout/denim_three/article/archivearticlebodytext.thtml
public_html/layout/denim_three/article/archivearticletext.thtml
public_html/layout/denim_three/article/articlebodytext.thtml
public_html/layout/denim_three/article/articletext.thtml
public_html/layout/denim_three/article/featuredarticlebodytext.thtml
public_html/layout/denim_three/article/featuredarticletext.thtml
public_html/layout/modern_curve/article/archivearticlebodytext.thtml
public_html/layout/modern_curve/article/archivearticletext.thtml
public_html/layout/modern_curve/article/articlebodytext.thtml
public_html/layout/modern_curve/article/articletext.thtml
public_html/layout/modern_curve/article/featuredarticlebodytext.thtml
public_html/layout/modern_curve/article/featuredarticletext.thtml
public_html/vendor/jquery-ui/jquery-ui-dialogoptions.js
public_html/vendor/jquery-ui/jquery-ui-slideraccess.min.js
public_html/vendor/jquery-ui/jquery-ui-timepicker-addon-i18n.min.js
public_html/vendor/jquery-ui/jquery-ui-timepicker-addon.min.js
system/classes/Database/DbMysqli.php
system/classes/Database/DbPgsql.php
system/vendor/google/recaptcha/src/ReCaptcha/RequestMethod/Socket.php
system/vendor/mobiledetect/mobiledetectlib/LICENSE
system/vendor/pear/net_dns2/tests/Tests_Net_DNS2_ParserTest.php
```

Not sure what is going on as these appear to be files that are new or changes since Geeklog 2.2.1sr1

**Comment by mystralkk** [comment]
Fixed with change set 0dec4dbf72af30c884333eecc98eb4a5eb2c10c6.
